package.json

```diff
@@ -6,6 +6,11 @@
   "engines": {
     "node": ">=8"
   },
+  "standard": {
+    "ignore": [
+      "*.jsx"
+    ]
+  },
```

Comment:
Is this neccessary? For me standard handles `jsx` just fine. Do I miss something?

Reply:
Yes, I couldnt figure out a better way than this unfortunately :/